phoebe/parameters/solver/optimizer.py

```diff
@@ -417,3 +417,73 @@ def differential_evolution(**kwargs):
     params += [IntParameter(qualifier='progress_every_niters', value=kwargs.get('progress_every_niters', 0), limits=(0,1e6), description='save the progress of the solution every n iterations.  The solution can only be recovered from an early termination by loading the bundle from a saved file and then calling b.import_solution(filename).  The filename of the saved file will default to solution.ps.progress within run_solver, or the output filename provided to export_solver suffixed with .progress.  If using detach=True within run_solver, attach job will load the progress and allow re-attaching until the job is completed.  If 0 will not save and will only return after completion.')]
 
     return ParameterSet(params)
+
+def differential_corrections(**kwargs):
+    """
+    Create a <phoebe.parameters.ParameterSet> for solver options for the
+    differential-corrections backend.
+
+    This only acts on LC and RV datasets.
+
+    Generally, this will be used as an input to the kind argument in
+    <phoebe.frontend.bundle.Bundle.add_solver>.  If attaching through
+    <phoebe.frontend.bundle.Bundle.add_solver>, all `**kwargs` will be
+    passed on to set the values as described in the arguments below.  Alternatively,
+    see <phoebe.parameters.ParameterSet.set_value> to set/change the values
+    after creating the Parameters.
+
+    For example:
+
+    ```py
+    b.add_solver('optimizer.differential_corrections')
+    b.run_solver(kind='differential_corrections')
+    ```
+
+    Parallelization support: differential_corrections does not support parallelization.  If
+    within mpi, parallelization will be handled at the compute-level (per-time)
+    for the <phoebe.parameters.compute.phoebe> backend.
+
+    Arguments
+    ----------
+    * `compute` (string, optional): compute options to use for the forward
+        model.
+    * `expose_lnprobabilities` (bool, optional, default=False): whether to expose
+        the initial and final lnprobabilities in the solution (will result in 1
+        additional forward model call).  Note that since `differential_corrections`
+        does not support priors, this is effectively the lnlikelihood.
+    * `continue_from` (string, optional, default='none'): continue the optimization
+        run from an existing solution by starting each parameter at its final
+        position in the solution.
```

Comment:
is this considered a preferred way to continue iterating? I can easily achieve the same using something like:
```
for iter in range(1, 10):
    b.run_solver(solver='solver_label', solution='solution_label', overwrite=True)
    # any logic to run sanity checks on the solution
    b.adopt_solution(solution='solution_label')
```

Reply:
Once you adopt, there is no easy way to "undo".  Using `continue_from` allows you to complete additional iterations without adopting (and is probably also quite a bit cheaper).  Granted, it's more useful for other optimizers where the information to continue is more involved than just starting positions.